Exercise_B/index.css

```diff
@@ -2,3 +2,27 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.blue {
+    background-color: blue;
+    float: left;
+}
+
+.red {
+    background-color: red;
+    position: fixed;
+    top: 30%;
+    left: 50%;
+}
+
+.green {
+    background-color: green;
+    float: right;
+}
+
+
+.box {
+    width: 100px;
+    height: 100px;
+}
```

Comment:
I like have you have used float left and right. To improve the positioning of the red box I would clear the floats in the CSS and use something like margin auto. 

On you red class only have the background colour and:
  clear: both;
  margin: auto;
  
  With this in mind you will need to alter your HTML swap lines 20 and 21 and it will work!